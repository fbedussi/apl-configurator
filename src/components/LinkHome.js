import React from 'react';

const LinkHome = ({text, url}) => {
    return <a className="linkHome" href={url}>{text}</a>;
};

export default LinkHome;
