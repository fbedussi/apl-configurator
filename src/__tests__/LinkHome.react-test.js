import React from 'react';
import LinkHome from '../components/LinkHome';
import renderer from 'react-test-renderer';

const props = {
  text: 'Link text',
  url: 'http://www.example.com'
};

test('LinkHone component', () => {
  const component = renderer.create(
    <LinkHome {...props} />
  );
  
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});