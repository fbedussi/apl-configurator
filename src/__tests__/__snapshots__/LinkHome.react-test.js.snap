// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkHone component 1`] = `
<a
  className="linkHome"
  href="http://www.example.com"
>
  Link text
</a>
`;
